ChangeLog

```diff
@@ -1,3 +1,12 @@
+2025-01-08  Simon Sobisch <simonsobisch@gnu.org>
+
+	* config/runtime.cfg: Add documentation for COB_LOAD_GLOBAL flag
+
```

Comment:
this goes to config/Changelog  (with a leading empty line), preferably with the original date from 2024